### ISSUE TITLE
Huber loss crashes with exit code 0xC0000005 in Windows

### ISSUE THREAD

**Issue by instigatorofawe** [keras-team/keras#14337]
I'm running an RTX 3080, running Python 3.8.5, keras 2.4.3, tensorflow 2.5 (nightly build), on CUDA 11.1 with CUDNN 8.0.5 on Windows 10 Enterprise.

keras.losses.Huber() is non-functional, resulting in crashes regardless of input. To replicate, simply run the following code:

```
import keras
import numpy

keras.losses.huber(numpy.array([1]), numpy.array([1]))
```

This does not result in any error messages, but Python crashes with the following message:
Process finished with exit code -1073741819 (0xC0000005).

This doesn't happen when using keras.losses.mean_squared_error, but does occur when using keras.losses.mean_absolute_error.

Full log of console output after running the above code:
```
2020-12-30 11:43:27.824608: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
2020-12-30 11:43:29.722794: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library nvcuda.dll
2020-12-30 11:43:29.749712: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1760] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.74GHz coreCount: 68 deviceMemorySize: 10.00GiB deviceMemoryBandwidth: 707.88GiB/s
2020-12-30 11:43:29.750122: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
2020-12-30 11:43:29.756671: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll
2020-12-30 11:43:29.756889: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll
2020-12-30 11:43:29.760374: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cufft64_10.dll
2020-12-30 11:43:29.761559: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library curand64_10.dll
2020-12-30 11:43:29.765439: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusolver64_10.dll
2020-12-30 11:43:29.768498: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusparse64_11.dll
2020-12-30 11:43:29.769216: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll
2020-12-30 11:43:29.769494: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1902] Adding visible gpu devices: 0
2020-12-30 11:43:29.770078: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-12-30 11:43:29.771117: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1760] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce RTX 3080 computeCapability: 8.6
coreClock: 1.74GHz coreCount: 68 deviceMemorySize: 10.00GiB deviceMemoryBandwidth: 707.88GiB/s
2020-12-30 11:43:29.771531: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
2020-12-30 11:43:29.771749: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll
2020-12-30 11:43:29.771957: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll
2020-12-30 11:43:29.772159: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cufft64_10.dll
2020-12-30 11:43:29.772355: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library curand64_10.dll
2020-12-30 11:43:29.772557: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusolver64_10.dll
2020-12-30 11:43:29.772755: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusparse64_11.dll
2020-12-30 11:43:29.772952: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll
2020-12-30 11:43:29.773179: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1902] Adding visible gpu devices: 0
2020-12-30 11:43:30.313746: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1300] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-30 11:43:30.313967: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1306]      0 
2020-12-30 11:43:30.314104: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1319] 0:   N 
2020-12-30 11:43:30.314395: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1446] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6703 MB memory) -> physical GPU (device: 0, name: GeForce RTX 3080, pci bus id: 0000:01:00.0, compute capability: 8.6)
Process finished with exit code -1073741819 (0xC0000005)
```

**Comment by instigatorofawe** [comment]
After some further testing, I think I've identified the (or at least one) source of the crashes.

keras.backend.abs() kills Python, regardless of what inputs are given
